### PR TITLE
[ui][new] Extending OAuth tests

### PR DESCRIPTION
**Pull request by mcada**
//test: @oauth

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java [hunk 1]
```diff
@@ -73,23 +69,24 @@
         SLACK,
         S3,
         FTP,
-        GOOGLE_MAIL
+        GOOGLE_MAIL,
+        SAP_Concur
```
Comment (avano) [inline ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java]: to upper case

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**